packages/api/package.json

```diff
@@ -6,23 +6,20 @@
   "engines": {
     "node": ">=16.19.0"
   },
-  "bin": {
-    "cumulus-api": "./bin/cli.js"
-  },
   "scripts": {
     "build-lambda-zips": "for x in $(ls dist); do (cd dist/${x} && rm -f lambda.zip && find . | xargs node ../../../../bin/zip.js lambda.zip && echo zipped ${x}); done",
     "build": "../../node_modules/.bin/webpack",
     "clean": "git clean -d -x -e node_modules -f",
     "link-api-bin": "sh ./bin/link-api-bin.sh",
     "package": "npm run prepare && npm run build && npm run link-api-bin && npm run build-lambda-zips",
     "prepare": "npm run tsc",
-    "reset-tables": "LOCALSTACK_HOST=localhost NODE_ENV=test FAKE_AUTH=true cumulus-api reset-tables",
-    "serve-dist-oauth": "LOCALSTACK_HOST=localhost NODE_ENV=test cumulus-api serve-dist",
-    "serve-dist-remote": "cumulus-api serve-dist",
-    "serve-dist": "LOCALSTACK_HOST=localhost NODE_ENV=test FAKE_AUTH=true cumulus-api serve-dist",
-    "serve-oauth": "LOCALSTACK_HOST=localhost LOCAL_ES_HOST=localhost NODE_ENV=test cumulus-api serve",
-    "serve-remote": "cumulus-api serve",
-    "serve": "LOCALSTACK_HOST=localhost LOCAL_ES_HOST=localhost NODE_ENV=test FAKE_AUTH=true cumulus-api serve",
+    "reset-tables": "LOCALSTACK_HOST=localhost NODE_ENV=test FAKE_AUTH=true ./bin/cli.js reset-tables",
+    "serve-dist-oauth": "LOCALSTACK_HOST=localhost NODE_ENV=test ./bin/cli.js serve-dist",
+    "serve-dist-remote": "./bin/cli.js serve-dist",
```

Comment:
A bunch of these don't appear to be working.    Still looking into it.

Reply:
It seems like some of these are rarely used. Or at least haven't been tested in a while. What I'm seeing is missing ENVs like `EARTHDATA_CLIENT_ID` (and related). After that's resolved it's more ES connection troubles.

Is it possible these commands can be deprecated? I'm not sure if the `serve-x` commands are used.

Reply:
Possible.  I'm not certain who to ask at this point.      The majority  of them are likely service for dashboard development/API testing.

Reply:
Discussed in the 5.3 architecture meeting. I think we can remove `reset-tables` here because it's not working and doesn't seem particularly useful. https://github.com/nasa/cumulus/pull/3354/commits/66f4581fb149d6cfcb291bf208142f42af7937ff

For the distribution API commands I think we need a separate effort. We have the ticket for looking at Elastic on M1 macs: https://bugs.earthdata.nasa.gov/browse/CUMULUS-3213

And after that I made another to work on the distribution API commands: https://bugs.earthdata.nasa.gov/browse/CUMULUS-3297

How does that seem? I wouldn't mind fixing it here but the Elastic issues are going to block everything after.